docs/source/guide/pec-2-use-case.myst

```diff
@@ -0,0 +1,33 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# When should I use PEC?
+
+## Advantages
+
+The main advantage of PEC is that, under the assumption of perfect gate tomography,
+it provides an unbiased estimation of expectation values.
+This means that, in the limit of many samples, the error mitigated expectation values
+converge to the ideal expectation values.
+
+
+## Disadvantages
+
+PEC is a noise-aware technique that can be applied only if
```

Comment:
Splitting hairs on this point again, but you can apply PEC whenever you want. I would rather say the PEC result converges to the noiseless results only if full tomographic knowledge of the hardware gates is known.